ui/app/src/components/NewExperimentNext/Step1.tsx

```diff
@@ -60,6 +61,33 @@ const useStyles = makeStyles((theme) => {
 
 const submitDirectly = ['pod-failure']
 
+interface TypeCardProp {
+  name: Env
+  handleSwitchEnv: any
```

Comment:
@g1eny0ung I have no idea how to fill the type here (so I put `any` for now 😢 ), do you have any specific suggestions?

Reply:
Should be `(env: Env) => () => void`.

Reply:
BTW for this situation, I usually copy from TS IntelliSense in VS Code then change it or not as appropriate. This makes it more convenient:

<img width="492" alt="image" src="https://user-images.githubusercontent.com/15034155/153334486-e86e8fb8-cd9a-4a70-b1e3-6e32bb2c4663.png">

Reply:
Got it! Fixed.